doc/source/fmpz_mat.rst

```diff
@@ -529,6 +529,11 @@ Matrix multiplication
     The matrices must have compatible dimensions for matrix multiplication.
     No aliasing is allowed.
 
+.. function:: void fmpz_mat_mul_blas(fmpz_mat_t C, const fmpz_mat_t A, const fmpz_mat_t B)
```

Comment:
Return value is void, but docs say it returns an int. Also some indication of when it can return 0 would be useful. I have assumed it only does so when BLAS is not available (which I need to check for anyway as I need to check the number of blas threads).